requirements.txt

```diff
@@ -1,4 +1,4 @@
-oslo.config
+oslo.config>3.18
```

Comment:
Does this really only work with `>3.18`. We only ship 3.17 in the product.

Reply:
The scheme parameter was introduced in [3.18](https://docs.openstack.org/oslo.config/latest/reference/types.html)

Reply:
Hm, seems so:
_Changed in version 3.18: Added schemes parameter_

Reply:
It seems that the "schemes" part was added as part of 3.18. I am not sure if we're able to update to 3.18 on Cloud 7 so we might have to leave that part out.